### PR TITLE
v1.0.4 release

### LINKED ISSUE #1173: Release v1.0.4

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

close #1173

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+- HostNetwork support ([#1202](https://github.com/pingcap/tidb-operator/pull/1202))
+- Support configuring sysctls for Pods and enable net.* ([#1175](https://github.com/pingcap/tidb-operator/pull/1175))
+- Add tidb-lightning support ([#1161](https://github.com/pingcap/tidb-operator/pull/1161))
+- Add new helm chart tidb-drainer to support multiple drainers ([#1160](https://github.com/pingcap/tidb-operator/pull/1160))
+
+## Detailed Bug Fixes and Changes
+
+- Add e2e scripts and simplify e2e jenkins file ([#1174](https://github.com/pingcap/tidb-operator/pull/1174))
+- Fix the pump/drainer data dir to avoid data loss caused by bad configuration([#1183](https://github.com/pingcap/tidb-operator/pull/1183))
+- Add e2e to init sql ([#1199](https://github.com/pingcap/tidb-operator/pull/1199))
```
Comment (weekface) [inline CHANGELOG.md]:
```suggestion
- Add init sql case to e2e ([#1199](https://github.com/pingcap/tidb-operator/pull/1199))
```

Comment (aylei) [reply CHANGELOG.md]: Not yet, I'm still working on stability test in IDC and hostnetwork test in AWS

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+
+We strongly recommend you to upgrade to `v1.0.4` if you are using any version under `v1.1.0.alpha`.
+
+`v1.0.4` introduces the following fixes comparing to `v1.1.0.alpha.3`:
+
+- HostNetwork support ([#1202](https://github.com/pingcap/tidb-operator/pull/1202))
+- Add permit host option for tidb-initializer job ([#779](https://github.com/pingcap/tidb-operator/pull/779))
+- Fix drainer misconfiguration in tidb-cluster chart ([#945](https://github.com/pingcap/tidb-operator/pull/945))
+- Set the default externalTrafficPolicy to be Local for TiDB service ([#960](https://github.com/pingcap/tidb-operator/pull/960))
+- Fix tidb-operator crash when user modify sts upgrade strategy improperly ([#969](https://github.com/pingcap/tidb-operator/pull/969))
```
Comment (lilin90) [inline CHANGELOG.md]: What does `sts` mean? Make it clearer if possible?
```suggestion
- Fix tidb-operator crash when users modify sts upgrade strategy improperly ([#969](https://github.com/pingcap/tidb-operator/pull/969))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+- Add maxFailoverCount limit to TiKV ([#976](https://github.com/pingcap/tidb-operator/pull/976))
```
Comment (lilin90) [inline CHANGELOG.md]:
```suggestion
- Add the `maxFailoverCount` limit to TiKV ([#976](https://github.com/pingcap/tidb-operator/pull/976))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,65 @@
...
+- Fix values file customization for tidb-operator on aliyun ([#983](https://github.com/pingcap/tidb-operator/pull/983))
+- Do not limit failover count when maxFailoverCount = 0 ([#978](https://github.com/pingcap/tidb-operator/pull/978))
+- Suspend ReplaceUnhealthy process for AWS tikv auto-scaling-group ([#1027](https://github.com/pingcap/tidb-operator/pull/1027))
```
Comment (lilin90) [inline CHANGELOG.md]:
```suggestion
- Suspend the `ReplaceUnhealthy` process for TiKV auto-scaling-group on AWS ([#1027](https://github.com/pingcap/tidb-operator/pull/1027))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,65 @@
...
+- Fix create_tidb_cluster_release variable does not work ([#1066](https://github.com/pingcap/tidb-operator/pull/1066)))
```
Comment (lilin90) [inline CHANGELOG.md]:
```suggestion
- Fix the issue that the `create_tidb_cluster_release` variable does not work ([#1066](https://github.com/pingcap/tidb-operator/pull/1066)))
```

**Comment by aylei** [comment]
Progress:
- deploy, upgrade, scale-in, scale-out and configuration change e2e case in EKS 1.13.11 with hostnetwork enabled, passed
- stabiliy-test, ran to apiserver crash case and still pending

**Pull request merged**